.github/workflows/publish-container.yml

```diff
@@ -42,11 +49,15 @@ jobs:
 
           # Push arch specific tag.
           docker tag "$TARBALL_TAG" "$DOCKER_ARCH_FULL_PATH"
-          docker push "$DOCKER_ARCH_FULL_PATH"
+          IMAGE_PUSH_OUT="$(docker image push "$DOCKER_ARCH_FULL_PATH")"
+
+          DIGEST="$(grep -E --only-matching 'digest: \S*' <<< "$IMAGE_PUSH_OUT" | cut -d ' ' -f 2)"
+          cosign sign --yes "${DOCKER_ARCH_FULL_PATH}@${DIGEST}"
```

Comment:
where does the signing actually take place?

Reply:
Sigstore does it, I believe.